sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb

```diff
@@ -201,19 +453,388 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Next Steps\n",
-    "You can see further examples of running a pipeline job [here](../)"
+    "Download the output of a pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(pipeline_job.name, download_path=\".\", output_name=\"best_model\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will get the ID of a best run and download the artifacts which goes with it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import yaml\n",
+    "\n",
+    "with open(os.path.join(\"named-outputs\", \"best_model\", \"MLmodel\"), \"r\") as f:\n",
+    "    ml_model = yaml.safe_load(f)\n",
+    "ml_model[\"run_id\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When we know the run ID of the best run, we can instantiate mlflow run object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "from mlflow.tracking.client import MlflowClient\n",
+    "\n",
+    "# Obtain the tracking URL from MLClient\n",
+    "MLFLOW_TRACKING_URI = ml_client.workspaces.get(\n",
+    "    name=ml_client.workspace_name\n",
+    ").mlflow_tracking_uri\n",
+    "\n",
+    "print(MLFLOW_TRACKING_URI)\n",
+    "mlflow.set_tracking_uri(MLFLOW_TRACKING_URI)\n",
+    "\n",
+    "\n",
+    "# Initialize MLFlow client\n",
+    "mlflow_client = MlflowClient()\n",
+    "\n",
+    "mlflow_best_run = mlflow_client.get_run(ml_model[\"run_id\"])\n",
+    "\n",
+    "print(\"Parent Run: \")\n",
+    "print(mlflow_best_run)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the artiracts for this run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create local folder\n",
+    "import os\n",
+    "\n",
+    "local_dir = \"./artifact_downloads\"\n",
+    "os.makedirs(local_dir, exist_ok=True)\n",
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
+    "    mlflow_best_run.info.run_id, \"outputs\", local_dir\n",
+    ")\n",
+    "print(\"Artifacts downloaded in: {}\".format(local_path))\n",
+    "print(\"Artifacts: {}\".format(os.listdir(local_path)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get metrics for each runs\n",
+    "In this code we list all runs, sharing the same parent run ID and ending by the underscore followed by the order number."
```

Comment:
Please replace with "In this code we list all child runs, i.e., all runs that share the same parent run ID and end in the underscore followed by the order number."

Reply:
Fixed